tests/common.bash

```diff
@@ -76,7 +76,7 @@ is_a_kata_runtime() {
 
 # Gets versions and paths of all the components
 # list in kata-env
-extract_kata_env() {
+function extract_kata_env() {
 	RUNTIME_CONFIG_PATH=$(kata-runtime kata-env --json | jq -r .Runtime.Config.Path)
 	RUNTIME_VERSION=$(kata-runtime kata-env --json | jq -r .Runtime.Version | grep Semver | cut -d'"' -f4)
 	RUNTIME_COMMIT=$(kata-runtime kata-env --json | jq -r .Runtime.Version | grep Commit | cut -d'"' -f4)
```

Comment:
Please use `sudo` to invoke `kata-runtime` command so that it works as expected.

Reply:
I don't think that's the case, @dborquez, as we're not changing anything on this path.

The error we faced was because I mistakenly removed a function that was metrics specific and was in the common.bash.

I introduced a new commit to the PR moving that function to metrics/gha-run.sh